pkg/platform/abstract/platform.go

```diff
@@ -373,34 +373,44 @@ func (ap *Platform) functionBuildRequired(createFunctionOptions *platform.Create
 }
 
 func (ap *Platform) GetProcessorLogsAndBriefError(scanner *bufio.Scanner) (string, string) {
-	var formattedProcessorLogs, briefErrorMessage string
+	var formattedProcessorLogs, briefErrorsLog string
+
+	var briefErrorsArray *[]string
+	briefErrorsArray = &[]string{}
+
+	// adds log to briefErrorsArray (prevents duplicate lines)
+	addToBriefErrorsLog := func(logLine string) {
+		for _, briefError := range *briefErrorsArray {
+			if logLine != "\n" && logLine == briefError {
```

Comment:
Please hold of on duplicate removal and let's design this better:
- duplicate by message only and not full log line
- may result in inconsistent log removal from different workers

I suggest we find a way to deterministically associate a line with a worker and then we can safely keep logs for just one worker for the `briefErrorMessage` and even tell the user which worker we're showing and which workers we're omitting